### PR TITLE
LPS-105422 Update dependencies

### PR DESCRIPTION
**Pull request by adolfopa**

### CODE REVIEW

**Comment by adolfopa** [comment]
ci:forward

File: modules/apps/layout/layout-seo-web/src/main/java/com/liferay/layout/seo/web/internal/util/AssetDisplayPageFriendlyURLMapper.java [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+import java.util.Locale;
+import java.util.Map;
+import java.util.Objects;
+
+import javax.servlet.http.HttpServletRequest;
+
+/**
+ * @author Adolfo Pérez
+ */
+public class AssetDisplayPageFriendlyURLMapper {
```
Comment (cgoncas) [inline modules/apps/layout/layout-seo-web/src/main/java/com/liferay/layout/seo/web/internal/util/AssetDisplayPageFriendlyURLMapper.java:40]: Hey @liferay-lima ,

Is there any way to move the mapper to the LayoutSEOManager? We need to use this code from the ContentDashboard module as well :).

Thanks!!

File: modules/apps/layout/layout-seo-web/src/main/java/com/liferay/layout/seo/web/internal/util/AssetDisplayPageFriendlyURLMapper.java [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+				assetRendererFactory.getAssetRenderer(assetEntry.getClassPK());
+
+			return new AssetDisplayPageFriendlyURLMapper(
+				(JournalArticle)assetRenderer.getAssetObject());
+		}
+
+		return new AssetDisplayPageFriendlyURLMapper(null);
+	}
+
+	public String getMappedAssetDisplayPageFriendlyURL(
```
Comment (cgoncas) [inline modules/apps/layout/layout-seo-web/src/main/java/com/liferay/layout/seo/web/internal/util/AssetDisplayPageFriendlyURLMapper.java:73]: Maybe the AssetDisplayPageFriendlyURLProvider could be helpful with this?

**Comment by cgoncas** [comment]
Hey @adolfopa !!

I just have added some comments to this pull,  sorry for meddling :P.

Thanks!!

File: modules/apps/layout/layout-seo-web/src/main/java/com/liferay/layout/seo/web/internal/util/AssetDisplayPageFriendlyURLMapper.java (hunk 2 above)
Comment (adolfopa) [reply modules/apps/layout/layout-seo-web/src/main/java/com/liferay/layout/seo/web/internal/util/AssetDisplayPageFriendlyURLMapper.java:73]: Mmm, but that does only work for the current locale, right? We need to process the URL for all available locales.

**Comment by adolfopa** [comment]
ci:forward

**Comment by austinchiang** [comment]
ci:test:relevant

**Comment by cgoncas** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by cgoncas** [comment]
ci:close

**Comment by cgoncas** [comment]
Continues in https://github.com/liferay-lima/liferay-portal/pull/519

**Pull request closed without merging**